assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -53,19 +55,20 @@ function LibraryProvider({ children }) {
     () => [
       MEDIA,
       MEDIA3P,
-      TEXT,
+      ...(tab === TEXT.id ? [TEXT] : [{ icon: TextIcon, id: 'text' }]),
```

Comment:
So we don't load up all the text sets right away and so we don't hold all of them in memory all the time, this logic will mount/unumount text pane based on whether the text tab is active or not.